### PR TITLE
os-release: change RELEASE_TYPE value from 'pre-release' to 'development' and break into paragraphs

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

**Comment by bluca** [comment]
FYI @AdrianVovk

File: man/os-release.xml [hunk 1]
```diff
@@ -331,21 +331,28 @@
 
           <listitem><para>A lower-case string (no spaces or other characters outside of 0-9, a-z, ".",
           "_", and "-"), describing what kind of release this version of the OS is. Known values follow:
-          <literal>stable</literal> is for normal releases of the system, suitable for production use.
-          Generally, stable releases become end-of-life soon after the next major stable release is out.
-          Examples include Fedora 40, Ubuntu 23.10, OpenSUSE Tumbleweed, and Arch Linux.
-          <literal>lts</literal> is for long term support releases of the system, suitable for production
+          </para>
+
+          <para><literal>stable</literal> is for normal releases of the system, suitable for production use.
```
Comment (YHNdnzj) [inline man/os-release.xml]: It's probably better to split this further into an `itemizedlist`?

File: man/os-release.xml [hunk 2]
```diff
@@ -331,21 +331,28 @@
...
+
+          <para><literal>development</literal> is for unstable versions of the system, unsuitable for production
           use, such as alpha, beta, or rolling unstable releases. Examples include Fedora Rawhide, Debian
-          Testing, Fedora 40 Beta, and GNOME OS Nightly.
-          <literal>experiment</literal> is for experimental builds of the system, created specifically to
+          Testing, Fedora 40 Beta, and GNOME OS Nightly.</para>
+
+          <para><literal>experiment</literal> is for experimental builds of the system, created specifically to
           test some work-in-progress feature. This is meant to be used in combination with <varname>EXPERIMENT=</varname>.
           If unset, or an unknown value, assume that the release is <literal>stable</literal>.</para>
```
Comment (AdrianVovk) [inline man/os-release.xml]: The last sentence isn't part of `experiment` and should be split out separately

**Comment by AdrianVovk** [comment]
Otherwise, LGTM

**Pull request merged**